src/io/flutter/run/daemon/DevToolsService.java

```diff
@@ -45,6 +53,19 @@ public static DevToolsService getInstance(@NotNull final Project project) {
 
   private DevToolsService(@NotNull final Project project) {
     this.project = project;
+
+    ApplicationManager.getApplication().executeOnPooledThread(() -> {
+      // TODO(helinx): Also use daemon method for later flutter versions where devtools.serve uses pub global run devtools
```

Comment:
later -> latter
instead of devtools.serve I assume you mean `flutter daemon`?

Reply:
I meant later here, i.e. flutter versions after the change in flutter/flutter. I hopefully clarified daemon/devtools.serve in the updated comments